src/role_reserver.js

```diff
@@ -41,6 +41,8 @@ roles.reserver.action = function(creep) {
     }
   }
 
+  creep.say('Reserver -> ' + creep.memory.routing.targetRoom, true);
```

Comment:
Why this? It just fills up more, and it gives information publicly.
This would be fine imo if it was not public

Reply:
Not necessary here.

IMHO we should prefer public over private, maybe `config` variable to disable public saying.
The reason for public is, that IMHO the TooAngel AI should be more seen as an NPC with a high level of transparency. Also the TooAngel users can debug others rooms, too.